nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -1,42 +1,26 @@
-- `CreateSchema`: Instructs the persister to create the table automatically, defaults to `true`
-
-The following settings are available for changing the behavior of subscription persistence through the `AzureSubscriptionStorageConfig` section:
 
-- `ConnectionString`: Sets the connection string for the storage account to be used for storing subscription information.  Defaults to `UseDevelopmentStorage=true` in Versions 6 and below, and defaults to `null` in Versions 7 and above.
-- `CreateSchema`: Instructs the persister to create the table automatically, defaults to `true`
-- `TableName`: Specifies the name of the table for storing subscriptions, defaults to `Subscription`.
+In NServiceBus.Persistence.AzureStorage Version 1 and above XML based configuration is no longer available. Configuring the behavior of the persister is done using the code configuration API.
 
-The following settings are available for changing the behavior of timeout persistence through the `AzureTimeoutPersisterConfig` section:
-
-- `ConnectionString`: Sets the connectionstring for the storage account to be used for storing timeout information.  Defaults to `UseDevelopmentStorage=true` in Versions 6 and below, and defaults to `null` in Versions 7 and above.
-- `TimeoutManagerDataTableName`: Sets the name of the table where the timeout manager stores it's internal state, defaults to `TimeoutManagerDataTable`
-- `TimeoutDataTableName`: Sets the name of the table where the timeouts themselves are stored, defaults to `TimeoutDataTableName`
-- `CatchUpInterval`: When a node hosting a timeout manager would go down, it needs to catch up with missed timeouts faster than it normally would (1 sec), this value  sets the catchup interval in seconds. Defaults to 3600, meaning it will process one hour at a time.
-- `PartitionKeyScope`: The time range used as partition key value for all timeouts. For optimal performance, this should be in line with the catchup interval so it should come to no surprise that the default value also represents an hour: `yyyyMMddHH`. Data in the table defined by `TimeoutDataTableName` will need to be migrated When modifying `PartitionKeyScope`.
-- `TimeoutStateContainerName`: Sets the name of the container where the timeout state is stored, defaults to `timeoutstate` - **Added in v7.0**
+### Configuration with Configuration Section
+In NServiceBus.Azure Version 6 and lower configuration can be performed using configuration sections and properties in the `app.config` file.
 
-For more information see [Configuring Azure Connection Strings](https://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/)
+snippet:AzurePersistenceFromAppConfig
 
 ### Configuration with Code
+Configuration of the persister via the code API is available to NServiceBus.Azure Version 6 and below as well as NServiceBus.Persistence.AzureStorage version one and above.
```

Comment:
Version 1

I want to verify with some of the other docs maintainers if we want to include version ranges when it's 1+